### PR TITLE
Implement a pipeline unit for filtering Python packages already present in the base

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/adviser/issues/1948

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: tests/sieves/test_thoth_s2i_packages.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+from thoth.adviser.pipeline_builder import PipelineBuilderContext
+from thoth.adviser.sieves import ThothS2IPackagesSieve
+from thoth.python import PackageVersion
+from thoth.python import Source
+
+from ..base import AdviserUnitTestCase
+
+
+class TestThothS2IPackagesSieve(AdviserUnitTestCase):
+    """Test filtering out packages based on symbols required."""
```
Comment (KPostOffice) [inline tests/sieves/test_thoth_s2i_packages.py:35]: Copy paste

**Review by KPostOffice — COMMENTED** [review]
LGTM. I really like this :+1:

File: tests/sieves/test_thoth_s2i_packages.py (hunk 1 above)
Comment (fridex) [reply tests/sieves/test_thoth_s2i_packages.py]: Thanks, adjusted in 6a9df1c4

**Pull request merged**